File: Two Sums/TwoSums.java
import java.util.*;

public class TwoSums {
    public static int[] twoSum(int[] nums, int target) {
        int i, j;
        int res[] = new int[2];
        int l = nums.length;
        System.out.println("length= "+l);
        for(i=0;i<=l-2;i++){
            for(j=i+1;j<=l-1;j++){
            	System.out.println("Checking"+nums[i]+" and "+nums[j]);
                if(nums[i]+nums[j]==target){
                    System.out.println("Success: "+i+" "+j);
                    
                    res[0] = i;
                    res[1] = j;	
                    
                }
            }
        }
        return(res);
    }

    public static void main(String[] args){
    	int[] a = {3, 2, 4};
    	int[] result = new int[2];
    	result = twoSum(a, 6);
    	// System.out.println(result[0]+result[1]);
    }
}

File: Two Sums/twosums.py

def twoSum(nums, target):
    res=[]
    i=0
    j=0
    print("Len"+str(len(nums)))
    for  i in range(0,len(nums)-1):
        for j in range(i+1,len(nums)):
            if nums[i]+nums[j]==target:
                res.append(i)
                res.append(j)
                print(str(i)+" "+str(j))
                return res

twoSum([2,  7, 11, 15], 9)